package.json

```diff
@@ -87,6 +87,14 @@
                     "default": null,
                     "description": "Specifies the path to the Phpactor binary"
                 },
+                "phpactor.executablePath": {
```

Comment:
maybe call this `phpBinaryPath`

Reply:
Suggested this to keep in line with Microsofts internal PHP linter naming...

I should definitely extend the readme with instructions tho.